src/connector/src/source/monitor/metrics.rs

```diff
@@ -38,10 +40,24 @@ impl SourceMetrics {
             registry
         )
         .unwrap();
+        let source_error_count = register_int_counter_vec_with_registry!(
+            "source_error_count",
+            "Source errors in the system, queryable by tags",
+            &[
+                "error_type",
+                "error_msg",
+                "executor_name",
+                "fragment_id",
+                "table_id"
+            ],
```

Comment:
why do we need fragment id here?
the same source belonging to different fragments will have the same errors.